meetings/Minutes_2018-02-20.md

```diff
@@ -0,0 +1,47 @@
+## Minutes - Jan 20, 2018
+
+1. **RQM Course**
+ * New material: simulations / bootstrapping & permutations (Lindsay & Luke), mixed effects modelling, time series analysis / simulating time series, practical quantitative stats (Lindsay - hypothesis testing, fit statistics)
+ * Look at content of the intro stats pre-req? Adjust datasets to fit what they could do?
+ * Ahmed has a copy of the textbook for the stats pre-req.
+ * Involvement:
+      * Lina: some lectures, some group support
+      * Lindsay: develop and teach stats lessons
+      * Ahmed: be TA, teach the R coding part, be the lead organizer
+      * Elliott: NA
+      * Joel: NA
+      * Madeleine: be TA, not lead, teach a few lectures (esp simulations and time series analysis)
+      * Nil: helping with organization, possibly come around 
+      * Sara: not officially involved, interested in seeing how it goes
+      * Ahmed's friend from EEB (James): knows the material, likes coding
+      * Having a couple of guest lecturers (possibly Lina) 
+ * Have another meeting soon and invite James from EEB
+ * Ask if we can have two course instructors (Lindsay and Ahmed or James) and two TAs (Madeleine and Lina or Ahmed)? 
+ * Otherwise, 4 TAs, equal division of hours: Lindsay, Madeleine, Ahmed, and James
+ * ACTION: Ahmed talk to James
+ * ACTION: Joel email Martin and Asher 
+ * Publishing a paper: publish on bioarxiv, or in this other journal that Data Carpentry publishes in - "International Journal of Digital Curation" (no publication cost)
+
+2. **SWC**
+ * Past workshop was generally well-received.
+ * A new workshop: one just for women.
```

Comment:
very good idea, I like that! You guys talk about collaborating with Ladies Who Code? (if that's the right name of the group)

Reply:
Ladies Learning Code? I actually taught for them last week, they'd definitely be good to talk to about this. Although their focus is less on programming for research.